File: VB-Examples/01_MIMIE Message/EmlToMHTML.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email

Namespace EmlToMHTML
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Dim mail As MailMessage = MailMessage.Load("..\..\..\..\..\..\..\Data\Sample.eml")
			mail.Save("LoadAndSave.mhtml", MailMessageFormat.Mhtml)
            MessageBox.Show("Completed")
		End Sub
	End Class
End Namespace

File: CS-Examples/01_MIMIE Message/EmlToMHTML.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;

namespace EmlToMHTML
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {            
            MailMessage mail = MailMessage.Load(@"..\..\..\..\..\..\..\Data\Sample.eml");
            mail.Save("LoadAndSave.mhtml", MailMessageFormat.Mhtml);
            MessageBox.Show("Completed");
        }
    }
}